pkg/apis/edgesite/v1alpha1/default.go

```diff
@@ -0,0 +1,166 @@
+					QueryConfigmapWorkers:             constants.DefaultQueryConfigMapWorkers,
+					QuerySecretWorkers:                constants.DefaultQuerySecretWorkers,
+					QueryServiceWorkers:               constants.DefaultQueryServiceWorkers,
+					QueryEndpointsWorkers:             constants.DefaultQueryEndpointsWorkers,
+					QueryPersistentvolumeWorkers:      constants.DefaultQueryPersistentVolumeWorkers,
+					QueryPersistentvolumeclaimWorkers: constants.DefaultQueryPersistentVolumeClaimWorkers,
+					QueryVolumeattachmentWorkers:      constants.DefaultQueryVolumeAttachmentWorkers,
+					QueryNodeWorkers:                  constants.DefaultQueryNodeWorkers,
+					UpdateNodeWorkers:                 constants.DefaultUpdateNodeWorkers,
+				},
+			},
+			Edged: &edgecoreconfig.Edged{
+				Enable:                      true,
+				NodeStatusUpdateFrequency:   constants.DefaultNodeStatusUpdateFrequency,
+				DockerAddress:               constants.DefaultDockerAddress,
+				RuntimeType:                 constants.DefaultRuntimeType,
+				NodeIP:                      "",
+				ClusterDNS:                  "",
+				ClusterDomain:               "",
+				EdgedMemoryCapacity:         constants.DefaultEdgedMemoryCapacity,
+				RemoteRuntimeEndpoint:       constants.DefaultRemoteRuntimeEndpoint,
+				RemoteImageEndpoint:         constants.DefaultRemoteImageEndpoint,
+				PodSandboxImage:             constants.DefaultPodSandboxImage,
+				ImagePullProgressDeadline:   constants.DefaultImagePullProgressDeadline,
+				RuntimeRequestTimeout:       constants.DefaultRuntimeRequestTimeout,
+				HostnameOverride:            constants.DefaultHostnameOverride,
+				RegisterNodeNamespace:       constants.DefaultRegisterNodeNamespace,
+				InterfaceName:               constants.DefaultInterfaceName,
+				DevicePluginEnabled:         false,
+				GPUPluginEnabled:            false,
+				ImageGCHighThreshold:        constants.DefaultImageGCHighThreshold,
+				ImageGCLowThreshold:         constants.DefaultImageGCLowThreshold,
+				MaximumDeadContainersPerPod: constants.DefaultMaximumDeadContainersPerPod,
+				CGroupDriver:                edgecoreconfig.CGroupDriverCGroupFS,
+			},
+			MetaManager: &edgecoreconfig.MetaManager{
+				Enable:                true,
+				ContextSendGroup:      metaconfig.GroupNameEdgeController,
+				ContextSendModule:     metaconfig.ModuleNameEdgeController,
+				PodStatusSyncInterval: constants.DefaultPodStatusSyncInterval,
+			},
+		},
+	}
+}
+
+// NewMinEdgeSiteConfig returns a common EdgeSiteConfig object
+func NewMinEdgeSiteConfig() *EdgeSiteConfig {
+	return &EdgeSiteConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       Kind,
+			APIVersion: path.Join(GroupName, APIVersion),
+		},
+		DataBase: &edgecoreconfig.DataBase{
+			DriverName: DataBaseDriverName,
+			AliasName:  DataBaseAliasName,
+			DataSource: DataBaseDataSource,
+		},
+		EdgeSite: &EdgeSiteDetail{
+			NodeName: "",
```

Comment:
nodeName here should have the same default value with HostnameOverride

Reply:
EdgeSiteDetail struct is useless, we can delete it